rsocket-core/src/main/java/io/rsocket/core/RSocketRequester.java

```diff
@@ -610,9 +611,9 @@ private void handleStreamZero(FrameType type, ByteBuf frame) {
         break;
       default:
         // Ignore unknown frames. Throwing an error will close the socket.
-        errorConsumer.accept(
-            new IllegalStateException(
-                "Client received supported frame on stream 0: " + frame.toString()));
+        if (LOGGER.isDebugEnabled()) {
+          LOGGER.debug("Requester received unsupported frame on stream 0: " + frame.toString());
+        }
```

Comment:
I wonder if this shouldn't be visible in production logs, perhaps `INFO` level?

Reply:
Interesting question. This is non-harmful thing that might happen. In general, it means that there is something wrong but it does not break the general work